examples/newrelic-express-apm/src/utils/response.js

```diff
@@ -58,7 +58,7 @@ class ResponseHandler {
       responseBody.data = data
     }
 
-    agent.recordMetric(
+    newrelic.recordMetric(
```

Comment:
You could put this metric also inside the ts-apm-sdk so that the user doesn't have to write any extra codes.

Reply:
Just as I mentioned [here](https://github.com/commercetools/commercetools-sdk-typescript/pull/861#discussion_r1914814445), I exported those util objects to allow for recording of other `custom` metrics. Don't you think this make sense?